.travis.yml

```diff
@@ -4,6 +4,8 @@ xcode_scheme: Tests
 osx_image: xcode9.1
 
 before_install:
+- gem uninstall --force -x -i /Users/travis/.rvm/gems/ruby-2.4.2@global xcodeproj
```

Comment:
Just curious: why do we provide a full to the gem? Wouldn't be enough to just say `gem uninstall xcodeproj` cause it seems that we're using default ruby version everywhere.

Reply:
Not sure, without the full path it failed and the travis log suggested using that path. On jenkins I didn't specify the full path.